visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.po.ts

```diff
@@ -0,0 +1,44 @@
+export class BlacklistPanelPageObject {
+	async checkExludedListAfterExclusion() {
+		const result = await page.evaluate(() => {
+			const element = document.querySelector("#excludedList")
+			const list = element.querySelectorAll("bdi")
+
+			if (list[0].innerHTML === "Add pattern via search or node context-menu" && list.length === 1) {
+				return false
+			}
+			const listContent: string[] = []
+			for (let i = 0; i < list.length; i++) {
+				listContent.push(list[i].innerHTML.trim())
+			}
+			return listContent.includes("*ts*") && listContent.includes("*html*")
+		})
+		return result
+	}
+
+	async checkExludedListAfterItemRemovalFromExclusionList() {
+		const excludeOption = await page.evaluate(() => {
+			const element = document.querySelector("#object-0")
+			return element
+		})
+
+		if (excludeOption) {
+			await expect(page).toClick("#object-0", { timeout: 3000 })
```

Comment:
I think you should change all toClick calls with this functions which should resolve your e2e test
https://github.com/MaibornWolff/codecharta/blob/78e62235d8114d6f5bdc3901c381301a5d76ef65/visualization/app/puppeteer.helper.ts#L25

Reply:
@Med-H Thanks will do.